src/backend/protocols/go.twitter/account.go

```diff
@@ -397,7 +410,30 @@ func (worker *AccountWorker) saveErrorState(infos map[string]string, err string)
 
 	// check failuresThreshold
 	if lastDate.Sub(firstDate)/time.Hour > failuresThreshold {
-		// TODO : disable remote identity and send nats message to identities_worker
+		// disable remote identity
+		err := worker.broker.Store.UpdateUserIdentity(&UserIdentity{
+			UserId: worker.userAccount.userID,
+			Id:     worker.userAccount.remoteID,
+		}, map[string]interface{}{
+			"Status": "inactive",
+		})
+		if err != nil {
+			log.WithError(err).Warnf("[saveErrorState] failed to deactivate remote identity %s for user %s", worker.userAccount.remoteID, worker.userAccount.userID)
```

Comment:
error instead of warning ?

Reply:
don't know, flow continues, it is not aborting the function.